client/grandpa-warp-sync/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+description = "A request-response protocol for handling grandpa warp sync requests"
+name = "sc-grandpa-warp-sync"
```

Comment:
Why create a new crate for this? I think it should just go in the existing grandpa crate. This also avoids having to expose some grandpa internals.

Reply:
It's better to have it as a standalone crate IMO as it only can be used when BABE is also being used afaik.

Reply:
As it exists now I think it's independent from the block production protocol, it's creating finality proofs from genesis to some latest finalized block regardless of how that block itself is validated (although it's true that we also need to fetch BABE data elsewhere). I won't press on moving this to the grandpa crate but can we rename it to `sc-finality-grandpa-warp-sync` so the naming is closer to the other crate?